Gem/Code/Source/Components/WasdPlayerMovementComponent.cpp

```diff
@@ -115,16 +115,6 @@ namespace MultiplayerSample
         // Update velocity
         UpdateVelocity(*wasdInput);
 
-        // Ensure any entities that we might interact with are properly synchronized to their rewind state
-        if (IsAuthority())
-        {
-            const AZ::Aabb entityStartBounds = AZ::Interface<AzFramework::IEntityBoundsUnion>::Get()->GetEntityLocalBoundsUnion(GetEntity()->GetId());
```

Comment:
I guess we don't need #include <AzFramework/Visibility/EntityBoundsUnionBus.h> in this file anymore?